lib/features/login/presentation/login_controller.dart

```diff
@@ -126,22 +129,26 @@ class LoginController extends GetxController {
 
   void _getOIDCConfiguration(Uri baseUri) async {
     loginState.value = LoginState(Right(LoginLoadingAction()));
-    await _getOIDCConfigurationInteractor.execute(baseUri)
+    if (_oidcResponse != null) {
```

Comment:
What about the opposite case if `_oidcResponse == null` ?

Reply:
this check is for safety programming only, we will not have the case this function is invoked with `_oidcResponse == null`